cloud/features/git-backed-storage.mdx

```diff
@@ -0,0 +1,41 @@
+---
+title: Git-Backed Storage
+description: Learn about the unique benefits of using Git-backed storage with Flipt Managed Cloud
+mode: "wide"
+---
+
+## What Is Git-Backed Storage?
+
+Flipt Managed Cloud is unique in that it uses your existing Git repositories for your feature flags and configurations. This means that your feature flag data is stored in a Git repository of your choosing, which allows you to use your existing Git workflow and tools.
+
+![Git-Backed Storage](/images/cloud/features/git/feature-flag-in-repository.png)
+
+Flipt Managed Cloud manages your feature flags in what we call [Environments](/cloud/concepts#environments).
+
+Each environment can be connected to a different Git repository, allowing you to manage your feature flags in different repositories for different environments.
+
+<Tip>
+  Environments can also share the same Git repository, as long as they are in
+  configured to use different directories.
+</Tip>
+
+## Why Use Git-Backed Storage?
+
+We believe that Git is the best way to manage configuration data. It allows you to use your existing Git workflow and tools, and it allows you to version your configuration data.
+
+We also believe that feature flags are a type of configuration data, and as such, they should be managed in the same way.
+
+<Note>
+  For more info on the benefits of using Git-backed storage, see the
+  [Benefits](/cloud/benefits#gitops-and-clickops-friendly) page.
+</Note>
+
+Git is also a powerful tool for collaboration. It allows you to review and merge changes, and it allows you to see the history of your changes.
+
+Finally, Git is a standard tool that many developers are already familiar with, which means that you can use your existing knowledge to manage your feature flags.
+
+![Flipt UI](/images/cloud/features/git/feature-flag-in-ui.png)
+
+Flipt Managed Cloud takes advantage of these benefits by using Git as the backend for your feature flags while also providing a managed service so that you don't have to worry about the underlying infrastructure.
+
+Flipt Managed Cloud also includes a friendly UI for managing your feature flags, so that you can easily change their state without having to know Git at all.
```

Comment:
Do you think it is worth leading with this? Something like this near the top (or maybe just a bit more embellishment here):

Git lives at the heart of Flipt managed cloud, but also you don't have to know or understand this to use it.
The UI is built to be consumed by non-Git users, but you can always break the glass and access the configuration directly in your own repository if you need to.